app/services/data_exports/data_storage.rb

```diff
@@ -103,10 +103,7 @@ def download_transcription_files(transcription, directory_path, single_transcrip
 
       metadata_file_regex = /^transcription_metadata_.*\.csv$/
       transcription.export_files.each do |storage_file|
-        is_transcription_metadata_file = metadata_file_regex.match storage_file.filename.to_s
-        unless is_transcription_metadata_file && !single_transcription_export
```

Comment:
Wanted to double check, so. now `download_file_from_storage` will run even if it `single_transcription_export` is true? is that right? (this is me trying to understand the changes)

Reply:
Holy double negatives--that line's hard to parse. `unless` is almost always confusing that way. The removed logic here was to check each file to see if it was the "metadata" file (one of the 4 different files attached to transcriptions via Active Storage). Before these changes: 

If the the attached file IS the metadata file AND you are doing a single transcription export, download it.
If that file IS the metadata file AND you're doing a multi-transcription export, don't.
If it's NOT the metadata file and it's a multi-transcription export, don't. 

Because in a multi-transcription export, the metadata files were all aggregated together in another step. If it was a single export, it was supposed to be included in the zip. Now, the metadata files are always downloaded for every transcription--they're still concatenated later, but from these per-transcription metadata files we're downloading here and now, rather than in the AggregateMetadataFileGenerator file where they were being pulled from the database and from Azure all over again.

Does that sound right? That's what this is supposed to do, lmk if you think the logic needs tweaking.

Reply:
Sounds right. And I think in that logic it would also download if it was a single transcription and it was not the metadata? 

So now (i.e. "new" change), is it because we know that it is the metadata (regardless of whether it is single or multiple transcription export), download it? right?

Again, I trust you know this better than I do, I'm trying to get a better understanding of the newer changes :) . Feel free merge whenever you feel ready.

Reply:
Yeah, since the AggregateMetadataFileGenerator doesn't run when it's a single transcription (via the `DataExports::DataStorage.zip_transcription_files`) so it needed to grab em all for that single one.

Goal for new changes is to download every file, regardless or what kind it is, so that the AggregateMetadataFileGenerator can use what's being downloaded at this point rather than repeating this whole song and dance again immediately after and roughly doubling the I/O cost to the db and Azure.